task/gimlet-inspector/src/main.rs

```diff
@@ -0,0 +1,140 @@
+                                SequencerRegistersResponseV0::SequencerTaskDead,
+                                None,
+                            ),
+                            Err(_) => {
+                                // The SeqError type represents a mashing
+                                // together of all possible errors for all
+                                // possible sequencer IPC operations. The only
+                                // one we _expect_ here is ReadRegsFailed.
+                                (SequencerRegistersResponseV0::SequencerReadRegsFailed, None)
+                            }
+                        };
+                        let mut len =
+                            hubpack::serialize(&mut tx_data_buf, &resp)
+                                .unwrap_lite();
+                        if let Some(t) = trailer {
+                            tx_data_buf[len..len + t.len()].copy_from_slice(&t);
+                            len += t.len();
+                        }
+                        meta.size = len as u32;
+                    }
+                }
+
+                // With the response packet prepared, we may need to attempt
+                // sending more than once.
+                loop {
+                    match net.send_packet(
+                        SOCKET,
+                        meta,
+                        &tx_data_buf[0..(meta.size as usize)],
+                    ) {
+                        Ok(()) => break,
+                        // If `net` just restarted, immediately retry our send.
+                        Err(SendError::ServerRestarted) => continue,
+                        // If our tx queue is full, wait for space. This is the
+                        // same notification we get for incoming packets, so we
+                        // might spuriously wake up due to an incoming packet
+                        // (which we can't service anyway because we are still
+                        // waiting to respond to a previous request); once we
+                        // finally succeed in sending we'll peel any queued
+                        // packets off our recv queue at the top of our main
+                        // loop.
+                        Err(SendError::QueueFull) => {
+                            sys_recv_closed(
+                                &mut [],
+                                notifications::SOCKET_MASK,
+                                TaskId::KERNEL,
+                            )
+                            .unwrap_lite();
+                        }
+                        // These errors should be impossible if we're configured
+                        // correctly.
+                        Err(SendError::NotYours | SendError::InvalidVLan) => {
+                            unreachable!()
+                        }
+                        // Unclear under what conditions we could se `Other` -
```

Comment:
```suggestion
                        // Unclear under what conditions we could see `Other` -
```

Reply:
Amusingly this comment was copied verbatim from udprpc, which contains the same typo. :-) Fixing.